### PR TITLE
Updates names and descriptions of common ballistics

### PR DESCRIPTION
**Pull request by PopeDaveThe3th**
## What Does This PR Do
title

## Why It's Good For The Game
It's lorein' time.

Our firearms will no longer be Great Value brand, and will now have some descriptions that tell you what they are and what they're chambered in.

Also done in preparation for Ballistics Extended Descriptions, coming soon(tm) to a codebase near you.

## Testing
- Loaded in
- Spawned guns
- Names and descriptions are updated

## Changelog
:cl:
tweak: Changed some ballistic weapon names and descs
/:cl:

### CODE REVIEW

File: code/modules/projectiles/guns/projectile/launchers.dm [hunk 1]
```diff
@@ -33,8 +33,8 @@
 	return
 
 /obj/item/gun/projectile/automatic/gyropistol
-	name = "gyrojet pistol"
-	desc = "A prototype pistol designed to fire self propelled rockets."
+	name = "\improper MX2000 gyrojet pistol"
+	desc = "A prototype pistol designed by Sunburst Heavy Industries, intended to fire self propelled rockets."
```
Comment (Fordoxia) [inline code/modules/projectiles/guns/projectile/launchers.dm:37]:
```suggestion
	desc = "A prototype pistol designed by Sunburst Heavy Industries, intended to fire self-propelled rockets."
```

File: code/modules/research/designs/weapon_designs.dm [hunk 2]
```diff
@@ -147,8 +147,8 @@
 
 //WT550 Mags
 /datum/design/mag_oldsmg
-	name = "WT-550 Auto Gun Magazine (4.6x30mm)"
-	desc = "A 20 round magazine for the out of date security WT-550 Auto Rifle"
+	name = "WT-550 PDW Magazine (4.6x30mm)"
+	desc = "A 20 round magazine for the WT-550 PDW"
```
Comment (Henri215) [inline code/modules/research/designs/weapon_designs.dm:151]:
```suggestion
	desc = "A 20 round magazine for the WT-550 PDW."
```

File: code/modules/research/designs/weapon_designs.dm [hunk 3]
```diff
@@ -157,8 +157,8 @@
 	category = list("Weapons")
 
 /datum/design/box_oldsmg
-	name = "WT-550 Auto Gun Ammo Box (4.6x30mm)"
-	desc = "A box of 20 rounds for the out of date security WT-550 Auto Rifle"
+	name = "WT-550 PDW Ammo Box (4.6x30mm)"
+	desc = "A box of 20 rounds for the WT-550 PDW"
```
Comment (Henri215) [inline code/modules/research/designs/weapon_designs.dm:161]:
```suggestion
	desc = "A box of 20 rounds for the WT-550 PDW."
```

File: code/modules/research/designs/weapon_designs.dm [hunk 4]
```diff
@@ -167,24 +167,24 @@
 	category = list("Weapons")
 
 /datum/design/box_oldsmg/ap_box
-	name = "WT-550 Auto Gun Armour Piercing Ammo Box (4.6x30mm AP)"
-	desc = "A box of 20 armour piercing rounds for the out of date security WT-550 Auto Rifle"
+	name = "WT-550 PDW Armour Piercing Ammo Box (4.6x30mm AP)"
+	desc = "A box of 20 armour piercing rounds for the WT-550 PDW"
```
Comment (Henri215) [inline code/modules/research/designs/weapon_designs.dm:171]:
```suggestion
	desc = "A box of 20 armour piercing rounds for the WT-550 PDW."
```

File: code/modules/projectiles/guns/projectile/automatic.dm [hunk 5]
```diff
@@ -273,7 +273,7 @@
 
 // Bulldog shotgun //
 /obj/item/gun/projectile/automatic/shotgun/bulldog
-	name = "\improper 'Bulldog' Shotgun"
+	name = "\improper AS-14 'Bulldog' Shotgun"
 	desc = "A compact, mag-fed semi-automatic shotgun for combat in narrow corridors, nicknamed 'Bulldog' by boarding parties. Compatible only with specialized 8-round drum magazines."
```
Comment (Fordoxia) [inline code/modules/projectiles/guns/projectile/automatic.dm:277]:
```suggestion
	desc = "A compact semi-automatic shotgun for combat in narrow corridors, nicknamed 'Bulldog' by boarding parties. Compatible only with specialized 8-round drum magazines."
```
Redundant mag-fed removed. The second sentence makes it obvious that it is fed from magazines.

File: code/modules/research/designs/weapon_designs.dm [hunk 6]
```diff
@@ -167,24 +167,24 @@
...
 	id = "box_oldsmg_ap"
 	materials = list(MAT_METAL = 6000, MAT_SILVER = 600)
 	build_path = /obj/item/ammo_box/wt550/wtap
 	category = list("Weapons")
 
 /datum/design/box_oldsmg/ic_box
-	name = "WT-550 Auto Gun Incendiary Ammo Box (4.6x30mm IC)"
-	desc = "A box of 20 armour piercing rounds for the out of date security WT-550 Auto Rifle"
+	name = "WT-550 PDW Incendiary Ammo Box (4.6x30mm IC)"
+	desc = "A box of 20 incendiary rounds for the WT-550 PDW"
```
Comment (Henri215) [inline code/modules/research/designs/weapon_designs.dm:179]:
```suggestion
	desc = "A box of 20 incendiary rounds for the WT-550 PDW."
```

File: code/modules/research/designs/weapon_designs.dm [hunk 7]
```diff
@@ -167,24 +167,24 @@
...
 	id = "box_oldsmg_ic"
 	materials = list(MAT_METAL = 6000, MAT_SILVER = 600, MAT_GLASS = 1000)
 	build_path = /obj/item/ammo_box/wt550/wtic
 	category = list("Weapons")
 
 /datum/design/box_oldsmg/tx_box
-	name = "WT-550 Auto Gun Uranium Ammo Box (4.6x30mm TX)"
-	desc = "A box of 20 uranium tipped rounds for the out of date security WT-550 Auto Rifle"
+	name = "WT-550 PDW Uranium Ammo Box (4.6x30mm TX)"
+	desc = "A box of 20 uranium tipped rounds for the WT-550 PDW"
```
Comment (Henri215) [inline code/modules/research/designs/weapon_designs.dm:187]:
```suggestion
	desc = "A box of 20 uranium tipped rounds for the WT-550 PDW."
```

**Review by Henri215 — APPROVED** [review]

**Review by S34NW — APPROVED** [review]

**Pull request merged**